README.md

```diff
@@ -1,2 +1,61 @@
 # William-Raga-Portfolio
-My first Portfolio project built with HTML, CSS and JavaScript. 
+
+My first Portfolio project built with HTML, CSS and JavaScript.
+
+## Built with
+
+- HTML
+- CSS
+
+## Setup for Basic HTML/CSS Project
+
+- Linter for HTML/CSS installed
+
+## Getting Started
+
+### Prerequisites
+
+- Text editor
+- Basic HTML/CSS Knowledge
+- JavaScript
+- Linters i.e. Webhint, stylelint and lighthouse
+- Git
+
+### Setup
+
+### Install
+
+### Usage
+
+### Deployment
+
+- [Linter](https://github.com/microverseinc/linters-config/tree/master/html-css)
+
+## Authors
+
+👤 **Author1**
+
+- Github: [@itsmraga-hub](https://github.com/itsmraga-hub)
+
+- Twitter: [@RagaMacharia](https://twitter.com/RagaMacharia)
+
+- Linkedin: [@itsmraga](https://www.linkedin.com/in/itsmraga/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse team
+- Inspiration
+
+## License
+
+- [MIT](https://github.com/itsmraga-hub/William-Raga-Portfolio/blob/master/LICENSE)
```

Comment:
[OPTIONAL]
Your MIT license should have a MIT.md in your repo and not a link outside your repo. Please create a MIT.md license file.